controller/equationAdditionController.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.post('/addition', (req, res) => {
+    const { value1, value2 } = req.body;
+
+    if (typeof value1 !== 'number' || typeof value2 !== 'number') {
+        res.status(400).json({ error: 'Os valores precisam estar no formato Number' });
+    } else {
```

Comment:
In that case, if the if condition is not met, you can let the code follow the workflow without the else statement